template/src/App.tsx

```diff
@@ -170,17 +246,26 @@ const TabbedApp = () => {
           prefixes: ['plaut-ro.github.io/luna', 'localhost'],
           config: {
             screens: {
-              Details: 'details',
-              Linking: 'linking',
-              Home: '*', // Fall back to if no routes match
+              NotFound: '*', // Match any unknown route with our NotFound screen
+              App: {
+                path: '', // omit '/App' in the browser URL bar
+                // @ts-ignore - TODO screens in nested nav do not seem to be typed correctly upstream?
```

Comment:
This smells awful and should probably block merge on this while I investigate it upstream but that won't happen for a few days. `ts-ignore` is always a sign you're doing something wrong, and the react-navigation types are fairly complicated but also they work well. I might just be missing some type augmentation that's necessary for it to work like for navigation routes: 

https://reactnavigation.org/docs/typescript#specifying-default-types-for-usenavigation-link-ref-etc
https://github.com/react-navigation/react-navigation/issues/9741#issuecomment-955947433

Reply:
Hey @mikehardy 👋🏼 Did you find a better way to handle the config type with this kind of nested navigation?